tests/pytest/core/test_views.py

```diff
@@ -37,6 +37,13 @@ def test_get(self, view, app_request, mocked_session_reset):
         assert response.status_code == 200
         mocked_session_reset.assert_called_once()
 
+    def test_form_valid(self, view, model_TransitAgency):
+        form = view.form_class(data=dict(select_transit_agency=model_TransitAgency.slug))
+
+        assert form.is_valid()
+        view.form_valid(form)
+        assert view.success_url
```

Comment:
We have the `model_TransitAgency` fixture in this test already, may as well assert that this `success_url` equals the expected value (`eligibility_index_url`).

Reply:
good call. i implemented this suggestion and cleaned up the commit history.